paparazzi-gradle-plugin/src/test/projects/verify-gif/src/test/java/app/cash/paparazzi/plugin/test/KeypadViewTest.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paparazzi.plugin.test
+
+import android.animation.ObjectAnimator
+import android.view.View
+import android.widget.LinearLayout
+import android.widget.TextView
+import app.cash.paparazzi.Paparazzi
+import org.junit.Rule
+import org.junit.Test
+
+class KeypadViewTest {
+  @get:Rule
+  val paparazzi = Paparazzi()
+
+  @Test
+  fun testViews() {
+    val keypad = paparazzi.inflate<LinearLayout>(R.layout.keypad)
```

Comment:
Do we need all this other stuff from the original `KeypadViewTest` or can we simplify it and perhaps avoid having this test fail because of a separate issue?

Reply:
Good call, will remove the snapshot's